.github/workflows/actions.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       with:
         java-version: 11
     - name: Build with Gradle
-      run: ./gradlew build
+      run: ./gradlew build buildMavenPlugin
```

Comment:
I tried and failed. `./gradlew build` is not a real task but some Gradle magic that delegates to all subproject tasks called `build`, so I cannot attached to this toplevel task directly, since it's not actually a task. Do you know how to do this?

Reply:
Some context: https://stackoverflow.com/a/15535887

There are multiple ways to solve this:
- Either define a build task explicitly in the project root and attach the build tasks of all subprojects to it (similar to the following)
   ```
var build = tasks.register('build')
subprojects { p ->
    build.dependsOn p.tasks.named('build')
}
````
- Move the actual maven plugin build from the project root to `teamscale-maven-plugin` by adding the project to the settings.gradle and creating a build.gradle file next to the pom.xml. In there just define the `buildMavenPlugin` named task as `build` so that it is picked up by Gradle when you execute `./gradlew build`. I would recommend this approach.